libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -405,6 +411,12 @@ bool AP_ICEngine::throttle_override(float &percentage)
         return true;
     }
 
+    // if THROTTLE_WHILE_DISARMED is set then we use the base_throttle, allowing the pilot to control throttle while disarmed
+    if (option_set(Options::THROTTLE_WHILE_DISARMED) && base_throttle > percentage) {
```

Comment:
disarmed check?

Reply:
again, don't need to check for how this is used